tactile_calibration/tactile_calibration_tools/CMakeLists.txt

```diff
@@ -104,7 +104,7 @@ catkin_python_setup()
 catkin_package(
 #  INCLUDE_DIRS include
 #  LIBRARIES tactile_calibration_tools
-#  CATKIN_DEPENDS rosbag rospy tactile_msgs
+CATKIN_DEPENDS tactile_msgs
```

Comment:
could we do a separate catkin_lint cleanup campain on the whole repo and not put that in this single PR ? there are many many errors in this repo with catkin_lint (your and my code) and this is maybe worth to do separately ?